gradle.properties

```diff
@@ -1,4 +1,4 @@
-quasarVersion=0.8.10_r3-SNAPSHOT
+quasarVersion=0.8.10_kryo5-SNAPSHOT
```

Comment:
To be discussed further.

My only concern here is if we end up wanting to regress to Kryo 4 for some reason in C4 (and hold J11 compilation target requirement as won't have J17 tweaks from Kryo 5 anymore) when doing J17 upgrade there (we're trying....).  Ideally we don't but that gets us in a version number pickle.

So maybe to give us wiggle room this should be 0.9 ? (and 0.8 remains J11 Quasar + Kryo 4, which ideally gets abandoned if adopting 0.9 goes well).

Reply:
The `_kryo5` is jus a _temporary_ tag so that I can publish something into Artifactory for people to play with. And Quasar is still compiled to Java 11 bytecode here, and so has no tweaks whatsoever for Java 17. So as far as I'm concerned, this upgraded version should work on both Java 11 _and_ Java 17.

Reply:
Do we have an answer to what the right tag to publish this under is?